script/build

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+RESULT=0
+
+# Fail if we're not using php71+
+if ! (php --version | grep -q "PHP 7.") ; then
+    echo "You're not using PHP 7.1+, please install it."
+    php --version
+    exit 1
+fi
+
+# setup and run composer
+if [ -n "$OAUTH" ]; then
+    composer config -g github-oauth.github.com $OAUTH
+fi
+
+composer --no-ansi --no-interaction install
+
+# Make sure the reports directory exists
+if [ ! -d "build/reports" ]; then
```

Comment:
We don't use "build/reports" any more in codebuil, feel free to remove lin 21, 22 and 23